admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -66,6 +66,46 @@ export default class CreateOrderPage {
 
     this.initListeners();
     this.loadCartFromUrlParams();
+
+    return {
+      refreshAddressesList: (refreshCartAddresses) => this.refreshAddressesList(refreshCartAddresses),
+      refreshCart: (refreshCart) => this.refreshCart(refreshCart),
+      search: (string) => this.customerManager.search(string),
+    };
+  }
+
+  /**
+   * Checks if correct addresses are selected.
+   * There is a case when options list cannot contain cart addresses 'selected' values
+   *  because those are outdated in db (e.g. deleted after cart creation or country is disabled)
+   *
+   * @param {Array} addresses
+   *
+   * @returns {boolean}
+   */
+  static validateSelectedAddresses(addresses) {
+    let deliveryValid = false;
+    let invoiceValid = false;
+
+    const keys = Object.keys(addresses);
+
+    for (let i = 0; i < keys.length; i += 1) {
```

Comment:
Add to adapt this for ESLint to be happy
@NeOMakinG Can you confirm I did well translating `for (const key in addresses) {` into this ?

Previous statement: https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js#L89

Reply:
Looks fine, even if initially I would use a .forEach, that's fine like this